src/lib/Components/Artist/Artworks/index.tsx

```diff
@@ -75,7 +75,7 @@ class Artworks extends Component<Props, State> {
       )
     }
   }
-
+  // tslint:disable-next-line:no-shadowed-variable
```

Comment:
React provides "Component" this function has "Component" - didn't think it was worth all the faffing to rename.